austrakka/utils/output.py

```diff
@@ -184,3 +184,15 @@ def create_response_object(message: str, message_type: str):
         RESPONSE_MESSAGE: message,
         RESPONSE_TYPE: message_type
     }
+
+
+def format_group_dto_for_output(data, out_format):
```

Comment:
Let's move this out of `output.py`; it's for lower level output and logging functions. I think `utils/helpers/groups.py` is a better place